packages/react-core/src/components/NumberInput/NumberInput.tsx

```diff
@@ -122,10 +131,15 @@ export const NumberInput: React.FunctionComponent<NumberInputProps> = ({
           </span>
         </Button>
         <input
```

Comment:
Would it be possible to just use the `<TextInput>` component here instead of recreating this functionality?

Reply:
It's looking like this should work fine... but:

- Without changing TextInput, snapshots will need to be updated to include `aria-invalid="false"` (previously `aria-invalid` would only come in if `true`) and a few `data-ouia` properties (component-id, component-type, and safe). Should I assume this is okay and just update the snaps?
- The NumberInput component `onChange` event only has `event` for a parameter while the TextInput component has `value, event`. To use TextInput in NumberInput without a breaking change.. I'm doing the following for the TextInput prop within NumberInput, is this okay?
`{...(onChange && { onChange: (v, e) => onChange(e) })}`

Reply:
@gitdallas that looks good tom me for the onchange.

Reply:
+1